iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/DigitalTwinServiceClientTests.java

```diff
@@ -68,8 +68,7 @@ public void setUp() throws URISyntaxException, IOException, IotHubException {
 
         this.deviceClient = createDeviceClient(protocol);
         deviceClient.open();
-        DigitalTwinAsyncClient asyncClient = new DigitalTwinAsyncClient(IOTHUB_CONNECTION_STRING);
-        digitalTwinClient = new DigitalTwinClient(asyncClient);
+        digitalTwinClient = DigitalTwinClient.createFromConnectionString(IOTHUB_CONNECTION_STRING);
```

Comment:
nit: only for the file name - shouldn't it be DigitalTwinClientTests?

Reply:
yes makes sense. Will change in a follow up PR.